src/renderer/js/component/address/view.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { clipboard } from "electron";
+import Link from "component/link";
+import classnames from "classnames";
+
+export default class Address extends React.PureComponent {
+  static propTypes = {
+    address: React.PropTypes.string,
```

Comment:
just curious, we just merged the react upgrade. I saw you changed all the React.PropTypes.etc... to just PropTypes.etc...

I believe this will break (there probably won't be a merge conflict because this is a new file. Maybe merge with master and re-check that this works after making necessary changes?

Reply:
Whoops. You're right. I'll update

Reply:
all good now 👍